python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -186,6 +186,88 @@ def __init__(
             fiberspectrograph=0.0,
         )
 
+    def get_calibration_configuration(self, name: str) -> dict[str, typing.Any]:
```

Comment:
Could we just incorporate this into `calculate_optimized_exposure_times`

Reply:
If you want to keep this separate, could we make the name of the function a bit more specific?